addons/main.go

```diff
@@ -130,6 +130,9 @@ func parseAddonFlags(addonFlags *addonFlags) {
 }
 
 func main() {
+	var webhookServerPort int
+	flag.IntVar(&webhookServerPort, "webhook-server-port", 9865, "The port that the webhook server serves at.")
```

Comment:
The other flags are defined at the top. Could we follow the same convention as other flags.